src/frequenz/client/dispatch/test/service.py

```diff
@@ -0,0 +1,264 @@
+# License: MIT
+# Copyright © 2024 Frequenz Energy-as-a-Service GmbH
+
+"""Mock classes for the dispatch api.
+
+Useful for testing.
+"""
+
+import dataclasses
+from dataclasses import dataclass
+from datetime import datetime, timezone
+
+import grpc
+import grpc.aio
+
+# pylint: disable=no-name-in-module
+from frequenz.api.dispatch.v1.dispatch_pb2 import Dispatch as PBDispatch
+from frequenz.api.dispatch.v1.dispatch_pb2 import (
+    DispatchCreateRequest as PBDispatchCreateRequest,
+)
+from frequenz.api.dispatch.v1.dispatch_pb2 import (
+    DispatchDeleteRequest,
+    DispatchGetRequest,
+    DispatchList,
+)
+from frequenz.api.dispatch.v1.dispatch_pb2 import (
+    DispatchListRequest as PBDispatchListRequest,
+)
+from frequenz.api.dispatch.v1.dispatch_pb2 import DispatchUpdateRequest
+from google.protobuf.empty_pb2 import Empty
+
+from frequenz.client.base.conversion_helper import to_datetime as _to_dt
+
+# pylint: enable=no-name-in-module
+from frequenz.client.dispatch import Client
+from frequenz.client.dispatch._internal_types import DispatchCreateRequest
+from frequenz.client.dispatch.types import Dispatch
+
+__all__ = ["FakeClient"]
+
+
+@dataclass
+class FakeService:
```

Comment:
You might want to update the commit message too, it still says `MockXxx`.

Reply:
You noticed that, eh? :eyes: